hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/ssl/SSLFactory.java

```diff
@@ -99,7 +101,13 @@
   public static final String SSL_SERVER_EXCLUDE_CIPHER_LIST =
       "ssl.server.exclude.cipher.list";
 
-  public static final String SSLCERTIFICATE = IBM_JAVA?"ibmX509":"SunX509";
+  public static final String KEY_MANAGER_SSLCERTIFICATE =
```

Comment:
we removed a public static final variable here. Search at Apache github, no Apache project use Hadoop's SSLFactory. SSLCERTIFICATE, so we're probably fine here.

Reply:
Yes. That's my understanding too. We will have to add checkstyle exceptions if we want to  keep the unused constants.